main.py

```diff
@@ -261,6 +261,42 @@
 ]
 
 
+def _update_numpy_base_dependencies(fn, record, instructions):
+    depends = record.get("depends", [])
+    updated = False
+    for i, dep in enumerate(depends):
+        if dep.startswith("numpy-base") and '==' not in dep and '<' not in dep:
+            depends[i] = dep + ",<2.0a0"
+            updated = True
+        elif dep == "numpy-base":
+            depends[i] = "numpy-base <2.0a0"
+            updated = True
+    if updated:
+        instructions["packages"][fn]['depends'] = depends
+    return updated
+
+
+def _update_numpy_dependencies(fn, record, instructions):
+    depends = record.get("depends", [])
+    updated = False
+    for i, dep in enumerate(depends):
+        if dep.startswith("numpy"):
```

Comment:
```suggestion
        if dep.split()[0] == "numpy":
```
Otherwise it will also match numpy-base, numpydoc...